functional/cypress-tests/cypress/integration/dashboards.js

```diff
@@ -161,11 +146,12 @@ describe('Dashboard switcher', () => {
   });
 
   it('Automatic switching works', () => {
-    const dashboardName = dashboardNameGen();
     let nextDashboardName;
 
-    cy.addDashboard(dashboardName, '8', '3');
-    cy.chooseDashboard(dashboardName);
+    addWidgetsDashboard(undefined, undefined, '3')
```

Comment:
Please explain me why you are using 'undefined' here...

Reply:
Let me answer with example:

```
// create function with default values
=> window.myFunction = function (x =1, y = 2, z = 3) { console.log(`x = ${x}, y=${y}, z=${z}`) }
ƒ (x =1, y = 2, z = 3) { console.log(`x = ${x}, y=${y}, z=${z}`) }

=> window.myFunction() // all defaults
out:  x = 1, y=2, z=3

=> window.myFunction(3,5,7) // no defaults
out: x = 3, y=5, z=7

=> window.myFunction(undefined,undefined,7) // default, default, non default
out: 1 x = 1, y=2, z=7

```
As you can see when we have a function that has some default values we can use undefined for arguments that we need default